tests/test_models.py

```diff
@@ -33,6 +40,15 @@ def __str__(self) -> str:
     uuid = 'e9e1df6b-c6d3-4a34-9227-c27056d596c6'
     assert Field.from_value(JavaUUID(uuid)) == Field(stringValue=uuid)
 
+    class PGObject:
```

Comment:
@macat 
I think `PGObject` is a typo. Could you change it to `PGobject`?

Reply:
Ups. sorry. fixed.